gms/versioned_value.hh

```diff
@@ -73,17 +71,14 @@ public:
 #endif
     }
 
-    versioned_value(sstring&& value, version_type version = version_generator::get_next_version()) noexcept
-        : _version(version), _value(std::move(value)) {
-    }
+    explicit versioned_value(sstring&& value, version_type version = version_generator::get_next_version()) noexcept
+        : _version(version), _value(std::move(value)) {}
 
     versioned_value() noexcept
         : _version(-1) {
     }
 
-    static sstring version_string(const std::initializer_list<sstring>& args) {
-        return fmt::to_string(fmt::join(args, std::string_view(versioned_value::DELIMITER_STR)));
-    }
+    static sstring version_string(const std::initializer_list<sstring>& args) { return fmt::to_string(fmt::join(args, versioned_value::DELIMITER_STR)); }
```

Comment:
Why put everything into single line like that? IMO it's less readable

Reply:
This was generally because editing the lines triggered the clang-format (that has now specific rules in the repo) and that caused making it format to a single line... I don't like it all that much either so I changed the clang-format file a bit to not perform these.

In particular, it could be also that the line length set there (currently 160) which makes the formatter use it in these case. Maybe the max line length could also be made shorter (e.g. 120), as the clang-format still allows to set up some penalties that might make the line be longer under some circumstances.

cc @kreuzerkrieg  - #20123